deployments/multus-daemonset-thick.yml

```diff
@@ -207,6 +214,9 @@ spec:
         - name: host-var-run
```

Comment:
I realize this is unrelated to this changeset, but... I'm wondering if this /var/run mount is obsolete. I think we used to use it for a crio reboot hack? Maybe we can remove in 4.0

Reply:
We still used to '/host/var/run/multus' for CNI cache. So we cannot remove it, but agreed that we could change to '/run'.
Let me add issue for that.

Reply:
Added as #829

Reply:
It is taken care of the PR.